diff --git a/app/streaming/input/input.cpp b/app/streaming/input/input.cpp
index fb11938b..5c0eb22e 100644
--- a/app/streaming/input/input.cpp
+++ b/app/streaming/input/input.cpp
@@ -47,7 +47,7 @@ SdlInputHandler::SdlInputHandler(StreamingPreferences& prefs, int streamWidth, i
     // Otherwise, we'll use raw input capture which is straight from the device
     // without modification by the OS.
     SDL_SetHintWithPriority(SDL_HINT_MOUSE_RELATIVE_MODE_WARP,
-                            prefs.absoluteMouseMode ? "1" : "0",
+                            prefs.absoluteMouseMode ? "1" : "1",
                             SDL_HINT_OVERRIDE);
 
 #if !SDL_VERSION_ATLEAST(2, 0, 15)
